bitrise.yml

```diff
@@ -327,7 +327,7 @@ workflows:
         title: Set OpenInFocus version numbers
     meta:
       bitrise.io:
-        stack: osx-xcode-13.3.x
+        stack: osx-xcode-14.1.x-ventura
```

Comment:
🙄 There are a few instances of `osx-xcode-13.3.x`. Should we use XCode 14.1 throughout?

Reply:
Yes, let's upgrade all stacks